docs/accessControl.md

```diff
@@ -198,6 +199,32 @@ To block students from viewing closed assessment scores, set `"showClosedAssessm
 
 The `showClosedAssessment` access rule restriction is only useful in conjunction with [disabling real-time grading](assessment.md#disabling-real-time-grading) and setting `"showClosedAssessment": false`.
 
+## Submittable assessments
+
+The `submittable` access rule restriction is useful for allowing students to see what assessments they have coming up. If `submittable` is set to `false`, students can see the assessment on the Assessments page, but they cannot create a new assessment instance or submit answers to questions. If an assessment is currently not submittable but will be in the future, students can see when the assessment will become submittable by looking at the `Available credit` column on the Assessments page. Assessments are submittable by default.
```

Comment:
I think this should also be used when releasing exams to students for post-exam review? Using this new restriction will ensure that a student who had never started the exam will be unable to start it during the post-exam review period.

Reply:
It depends on what the "post-exam review period" looks like. If `submittable` is set to false, students who have not started the exam will not be able to see the questions at all. That's fine if the instructor only wants the students who did the exam to be able to see the questions. On the other hand, if the instructor wants to allow everyone to see the exam questions, they should *not* set `submittable` to false. Instead, they should set `credit: 0` (or omit the `credit` field so that it defaults to 0) in the access rules. In the end, it depends on whether or not the instructor wants to let everyone see the questions.

Reply:
You describe the situation exactly correctly, including the current practice. I think we should recommend that people change this current practice to instead use `"submittable": false` when returning exams. It would be great to specify this in the docs, both in the section about the `submittable` restriction and also here: https://prairielearn.readthedocs.io/en/latest/faq/#how-do-i-give-students-access-to-view-their-exams-after-they-are-over

Reply:
Documentation has been added